src/main/java/com/actiontech/dble/cluster/ClusterLogic.java

```diff
@@ -856,7 +857,6 @@ public static void syncShardingXmlToLocal(String shardingConfig, XmlProcessBase
 
         //the config Value in ucore is an all in one json config of the sharding.xml
         Shardings sharding = ClusterLogic.parseShardingJsonToBean(gson, shardingConfig);
-        ClusterLogic.writeMapFileAddFunction(sharding.getFunction());
```

Comment:
why romoved?

Reply:
the file has already been written , this is not required